src/lib/core/BUILD.gn

```diff
@@ -84,6 +84,20 @@ source_set("chip_config_header") {
   allow_circular_includes_from = [ "${chip_root}/src/ble:ble_config_header" ]
 }
 
+source_set("chip_error") {
```

Comment:
nit: maybe just `error` since chip is a fillter word like matter in our project

Reply:
Can do.